rust/core/src/driver_manager.rs

```diff
@@ -184,6 +205,41 @@ impl ManagedDriver {
         Ok(ManagedDriver { inner })
     }
 
+    /// Load a driver either by name, filename, path, or via locating a toml manifest file.
+    /// The LoadFlags control what directories are searched to locate a manifest.
+    pub fn load_from_name(
+        name: impl AsRef<OsStr>,
+        entrypoint: Option<&[u8]>,
+        version: AdbcVersion,
```

Comment:
Maybe you can add something to the function docs about these args?

Reply:
added to the docs